src/app/Views/user.html

```diff
@@ -11,6 +11,12 @@
 </div>
 
 <div class="menu-holder"></div>
+<div class="menu-button" *ngIf="info && areFriends" (click)="talk(conversationId)">
```

Comment:
Any reason to put it here?

Reply:
I think the message button is more commonly used than the delete button, so it's should move to the top. And delete button ususally located in the bottom.

Reply:
Really great idea!